src/components/Post/PostHeader.jsx

```diff
@@ -136,6 +137,10 @@ const PostHeader = ({ postId, user, postData, postHasImages, timestamp }) => {
         <p className="post__time">{time}</p>
       </Link>
       <div className="social__icon__last">
+       
+        {
+          
+          (user?.uid == username || location.pathname == "/dummygram/" || location.pathname == "/dummygram/favourites" || location.pathname == "/dummygram/search") && 
```

Comment:
you still didn't follow what I asked in my review. I asked to add this condition to only hide visit profile option and any relevant option.

Reply:
@narayan954  bro i think it working fine 
<img width="503" alt="image" src="https://github.com/narayan954/dummygram/assets/68702055/24d1f529-03ca-462f-bdea-607924011c9c">

<img width="372" alt="image" src="https://github.com/narayan954/dummygram/assets/68702055/c1173b65-d2bc-4dab-8e50-e67885da9758">

Reply:
images from search pages and profile pages
in profile pages there is no other option on view profile is there that's why hidding the three dot's

Reply:
> @narayan954 bro i think it working fine <img alt="image" width="503" src="https://user-images.githubusercontent.com/68702055/256180572-24d1f529-03ca-462f-bdea-607924011c9c.png">
> 
> <img alt="image" width="372" src="https://user-images.githubusercontent.com/68702055/256180670-c1173b65-d2bc-4dab-8e50-e67885da9758.png">

it's on profile page bro, I think you're showing saved page broooo

Reply:
> images from search pages and profile pages in profile pages there is no other option on view profile is there that's why hidding the three dot's

there's these options in owner's profile
![image](https://github.com/narayan954/dummygram/assets/77617189/b1dc1618-e431-49f1-83c1-b0c53017e2f0)
and yes on other's profile, there're no other option than that, but we just need to remove this option, so it works globally. 
![image](https://github.com/narayan954/dummygram/assets/77617189/461c21c0-7695-4c30-a57d-8d6e31033c30)

Reply:
I think maybe remove this ... at once as you're saying would be better to remove the complications in a single move, so I'm allowing your changes... sorry for the inconvenience